### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SichVictor**
https://sichvictor.github.io/layout_stars/
https://sichvictor.github.io/layout_stars/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(/images/star-active.svg);
+}
```
Comment (sTorba24) [inline src/style.css:39]: if selectors has same style, you can write them splited by comma 
.stars--1 .stars__star:nth-child(-n + 1),
.stars--1 .stars__star:nth-child(-n + 2)
etc...

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
+}
```
Comment (sTorba24) [inline src/style.css:3]: don't reset margins and paddings for `*` selector. Use tags directly

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost good! lets improve a little

**Comment by SichVictor** [comment]
Hello made changes

File: src/index.html [hunk 3]
```diff
@@ -7,7 +7,53 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
```
[the remark this answers is not in the document]
Comment (SichVictor) [reply src/index.html:10]: understand

File: .linthtmlrc.json
[the remark this answers is not in the document]
Comment (SichVictor) [reply .linthtmlrc.json:1]: ok

**Pull request closed without merging**